### PR TITLE
feat: allow password to be provided via CLI

### PR DESCRIPTION
**Pull request by bcoe**
noticed that password can't be provided via the CLI, definitely an oversight.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,19 @@ tools.
...
 ```bash
-npm install --save pgtools
+npm install --save -g pgtools
 ```
 
-# Example
+## CLI Example
+
+```bash
+createdbjs my_awesome_db --user=admin --pasword=admin
```
Comment (olalonde) [inline README.md:26]: Should be `password` not `pasword`

**Comment by olalonde** [comment]
Actually, the original `createdb` doesn't allow you to pass a  password as a CLI flag if I recall correctly... but I don't mind adding it to createdbjs...

File: README.md (hunk 1 above)
Comment (bcoe) [reply README.md:26]: whops, apologize for not catching that; was excited to get something released for work that uses this; fixing now.

**Comment by bcoe** [comment]
@olalonde looks like the original tool takes `--password`, but then prompts the user for the pass; definitely seems more convenient for us to just let it be passed in. My use-case is having a docker container that automatically creates a DB and runs migrations on boot.

**Comment by olalonde** [comment]
Sure, no worries. The official way to do this with `createdb` is something like `PGPASSWORD=somepass createdb ...`. I just remember this detail because I remember being frustrated at not being able to automate something with createdb haha :)

**Pull request merged**